packages/components/src/Table/_docs/Table.mdx

```diff
@@ -23,31 +23,61 @@ A table displays rows of data, including all data in a set, making it efficient
 
 <Canvas of={TableStories.Playground} />
 
-## API
+## TableContainer API
```

Comment:
Keeping this as a one pager but I've sectioned the props descriptions to each table component for easier digestion